### PR TITLE
torbrowser: 11.0 -> 11.0.2

### PR DESCRIPTION
**Pull request by jpdoyle**
###### Motivation for this change

Updating `torbrowser` to `11.0.2`. I don't know how to test building for i686 so I left a stubbed-out `sha256` and a TODO

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix [hunk 1]
```diff
@@ -97,15 +97,16 @@ let
...
     };
 
     i686-linux = fetchurl {
       urls = [
         "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux32-${version}_${lang}.tar.xz"
         "https://tor.eff.org/dist/torbrowser/${version}/tor-browser-linux32-${version}_${lang}.tar.xz"
       ];
-      sha256 = "07v1ca66a69jl238qdq81mw654yffrcyq685y4rvv8xvx11fnzzp";
+      # TODO
+      sha256 = "1111111111111111111111111111111111111111111111111111";
```
Comment (jyooru) [inline pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix:109]:
```bash
$ nix-prefetch-url "https://dist.torproject.org/torbrowser/11.0.2/tor-browser-linux32-11.0.2_en-US.tar.xz"
path is '/nix/store/6qi7j2i0wv9n1dhczzqbn77k5raw67i4-tor-browser-linux32-11.0.2_en-US.tar.xz'
1blp4z9rmnnsvl3bk0ajdccvpzfshnpyijjfiqb9ma02qw2z0gff
```
```suggestion
      sha256 = "1blp4z9rmnnsvl3bk0ajdccvpzfshnpyijjfiqb9ma02qw2z0gff";
```

**Review by jyooru — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix (hunk 1 above)
Comment (jpdoyle) [reply pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix]: thanks! I didn't know that gadget

**Comment by jyooru** [comment]
Result of `nixpkgs-review pr 149985` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tor-browser-bundle-bin</li>
  </ul>
</details>

**Review by jyooru — APPROVED** [review]
Tested `./results/tor-browser-bundle-bin/bin/tor-browser`, I can browse Tor.

**Comment by bobby285271** [comment]
#149606

**Pull request closed without merging**